region/crateria/east.json

```diff
@@ -1329,13 +1382,41 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
+                    "HiJump",
+                    "canWalljump"
```

Comment:
With hjb, you don't need a walljump. A well-timed jump from the furthest platform on the right can get you clear out of the water and onto the middle platform.